CMakeLists.txt

```diff
@@ -385,14 +385,10 @@ IF(NOT CYCLUS_DOC_ONLY)
     # set core version, one way or the other
     IF(NOT "${CORE_VERSION}" STREQUAL "")
         SET(core_version "${CORE_VERSION}")
-    ELSE(NOT "${CORE_VERSION}" STREQUAL "")
-        EXECUTE_PROCESS(COMMAND git describe --tags
-                        OUTPUT_VARIABLE core_version
-                        OUTPUT_STRIP_TRAILING_WHITESPACE)
     ENDIF(NOT "${CORE_VERSION}" STREQUAL "")
     IF("${core_version}" STREQUAL "")
-        MESSAGE(WARNING "Unable to read current core version, falling back to previous version. Core version will be set to -1")
-        SET(core_version "-1")
+        MESSAGE("No core version was specified... using version ${CMAKE_PROJECT_VERSION}")
+        SET(core_version ${CMAKE_PROJECT_VERSION})
```

Comment:
Do we typically use a patch version number in the `dbtypes`.  Should this just be MAJOR.MINOR? (at least to match past conventions)